values.yaml

```diff
@@ -37,23 +47,32 @@ octopus:
   # Azure AKS uses azurefile for this ReadWriteMany volumes.
   # Google uses Filestore for ReadWriteMany volumes: https://cloud.google.com/filestore/docs/accessing-fileshares
   # Amazon uses EFS for ReadWriteMany volumes: https://docs.aws.amazon.com/eks/latest/userguide/efs-csi.html
-  storageClassName: ""
+  storageClassName: "test"
+
   # When replica count is 1, this value defines the storage access type. When replica count is greater than 1, the storage more is always ReadWriteMany.
-  storageAccessMode: ReadWriteOnce
+  storageAccessModeOneReplica: ReadWriteOnce
```

Comment:
The `storageAccessModeOneReplica` and `storageAccessModeManyReplicas` values aren't used in the templates, and so can be removed from the values.

Reply:
Done